govuk_chat_evaluation/rag_answers/evaluate.py

```diff
@@ -1,31 +1,123 @@
+import os
 from pathlib import Path
+from typing import cast
+from functools import cached_property
+import pandas as pd
 
 from deepeval import evaluate as deepeval_evaluate
-from deepeval.metrics import GEval
-from deepeval.test_case import LLMTestCaseParams
+from deepeval.metrics import BaseMetric
 
+from .deepeval_evaluate import run_deepeval_evaluation, convert_deepeval_output_to_evaluation_results
 from ..file_system import jsonl_to_models
-from .data_models import EvaluationTestCase
+from .data_models import EvaluationTestCase, EvaluationConfig, EvaluationResult
 
 
+DEEPEVAL_EVAL_PARAMETERS = {
+    "print_results": False,
+    "show_indicator": True,
+    "max_concurrent": 40,
+    "throttle_value": 5,
+    "use_cache": True,
+    "ignore_errors": True,
+}
+
 # would expect we need to pass config object through if that has metrics configuration
-def evaluate_and_output_results(_output_dir: Path, evaluation_data_path: Path):
-    models = jsonl_to_models(evaluation_data_path, EvaluationTestCase)
-    run_deepeval_evaluation(models)
+def evaluate_and_output_results(
+    output_dir: Path, 
+    evaluation_data_path: Path, 
+    evaluation_config: EvaluationConfig
```

Comment:
I expect this won't be an instance of EvaluationConfig any more and would be an instance of Config 

(I also expect we'll have to move Config to be in the data_models.py file to avoid a cyclic dependency)

Reply:
Implemented both these changes.